### PR TITLE
NavBar visual improvements

### PR DESCRIPTION
**Pull request by arse0039**
Messed around with the navBar colors and font. 
Also added some icons and hover effects. Not sure if I like it or not as it may be too busy.

### CODE REVIEW

**Comment by davidclaphan** [comment]
I like the icons/animation, maybe back to the orange color? My vote is add in whatever animations we want to start, they're more fun for the user, and we can dial it back if things feel too busy later.

**Comment by arse0039** [comment]
No longer a draft. I landed on a nav bar that looks pretty solid with minimal animations. The icons were too busy, imo, so I opted for a simpler effect in order to bring focus to the main content.

File: frontend/src/constants/data.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { BsPersonArmsUp } from "react-icons/bs";
+import React, { ReactElement } from "react";
+
+export interface NavLink {
+    name:string;
+    icon:ReactElement;
+}
+
+export const navLinks:NavLink[] = [
+    {name:'Home', icon: <HiOutlineHome/>},
```
Comment (davidclaphan) [inline frontend/src/constants/data.tsx]: Are the icons still active?

**Comment by davidclaphan** [comment]
Clean, looks great

**Review by davidclaphan — DISMISSED** [review]

**Review by davidclaphan — DISMISSED** [review]

**Review by davidclaphan — APPROVED** [review]

**Pull request merged**